### PR TITLE
Text visibility in FSM

### LINKED ISSUE #308: Text not visible in Dark mode
Make the text visible in the dark mode.

b
a

### Are you Working on this Issue ??
**NO. Anyone can take and solve this issue.**

### PR DESCRIPTION
**Pull request by nileshdas**
Fixed the text visibility arrow labels in dark mode.

Fixes #308 

#### Changes done:
- [x] Changed the text fill to accommodate both light and dark mode.

#### Screenshots
Screenshot (131)
Screenshot (132)

#### ✅️ By submitting this PR, I have verified the following

- [x] Checked to see if a similar PR has already been opened 🤔️
- [x] Reviewed the contributing guidelines 🔍️
- [x] Sample preview link added (add the link(s) for all the pages changed/updated from the checks tab after checks complete)
- [x] Tried Squashing the commits into one

### CODE REVIEW

**Comment by nileshdas** [comment]
@abhinavsri360 I noticed that but I don't know how to remove the commit now. Can you suggest something?

**Comment by abhinavsri360** [comment]
@nileshdas this might help
https://www.clock.co.uk/insight/deleting-a-git-commit

**Comment by nileshdas** [comment]
@abhinavsri360 Will this commit 103d0f2a03ef3649f241dda70bdf8ee74165bcbd do the job?

**Comment by abhinavsri360** [comment]
@nileshdas look at the deploy link for these changes here: https://5edbaa071ecddf00081a607c--tender-ardinghelli-ebb79f.netlify.app/docs/binary.html

File: assets/css/dark-mode-preview.scss [hunk 1]
```diff
@@ -38,7 +38,6 @@
 @import "./svg_paths";
 @import "./quiz";
```
Comment (YashKumarVerma) [inline assets/css/dark-mode-preview.scss:40]: maybe this can be done when this file is edited

**Comment by nileshdas** [comment]
Preview link : https://deploy-preview-411--tender-ardinghelli-ebb79f.netlify.app/docs/binary.html

**Comment by YashKumarVerma** [comment]
Thanks a lot for your contributions.

File: assets/css/dark-mode-preview.scss (hunk 1 above)
Comment (nileshdas) [reply assets/css/dark-mode-preview.scss:40]: Will take care of it from next time surely.

**Pull request merged**